src/client.cpp

```diff
@@ -206,7 +206,7 @@ void Client::window_focus() {
 
     if (this != lastfocus) {
         /* FIXME: this is a workaround because window_focus always is called
-         * twice.  see BUGS for more information
+         * twice.
```

Comment:
Not sure what do here? Should we fill in the old info from BUGS into this comment?

Reply:
Right, we could insert
```
window_focus is called twice when switching desktops (before mapping
windows and after mapping them).
```
I'm also planning to clean the focus-mess up and move all the input focus handling to the X11-specific classes.